AlfrescoApp/Model/Managers/AccountManager.m

```diff
@@ -23,7 +23,6 @@
 #import "AccountCertificate.h"
 #import "AlfrescoProfileConfig.h"
 #import "AppConfigurationManager.h"
```

Comment:
With the inclusion of the protocol headers in the .h file, shouldn't this import be removed?